src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+
+    public Ball getRandomBall() {
+        int value = random.nextInt(100);
+        ColorSupplier color = new ColorSupplier();
```

Comment:
Consider making the `color` variable `final` since it's not changed after initialization. Also, it might be better to rename the variable to `colorSupplier` to avoid confusion with the actual color and move it to class variable as done for Random. 🎨